### ISSUE TITLE
try to reproduce the small aflw error 

### ISSUE THREAD

**Issue by simin75simin** [Ascend-Research/HeadPoseEstimation-WHENet#21]
so i was trying to reproduce whenet-v using the same network structure, preprocessor, etc. i could only get around 30 yaw loss and around 10 pitch and roll loss for AFLW2000, even after filtering out angles greater than 99 degrees. i saw in your paper that whenet-v was trained on 300w and got an MAE on AFLW2000 below 5. 
i used hopenet's preprocessor for 300W and a batch size of 16, 1e-5 learning rate, same network structure, 25 episodes to train on 300W, etc. what should i check / change to reproduce the small AFLW error?

also in your opinion can be model be further compressed to say like a few hundred kilobytes (assuming more training data, etc.) with a small loss of accuracy in practice? thanks.
